lib/ssz_ex.ex

```diff
@@ -15,6 +15,123 @@ defmodule LambdaEthereumConsensus.SszEx do
   @bits_per_byte 8
   @bits_per_chunk @bytes_per_chunk * @bits_per_byte
   @zero_chunk <<0::size(@bits_per_chunk)>>
+  @zero_hashes <<0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
```

Comment:
Can we compute this at compile-time? That way it's clear what this is, and how it can be extended if needed.

Reply:
How can I go about this?

Reply:
First of all, how did you get this string? Just to clarify, "compile-time" in Elixir is simple: just save it in a module attribute (`@some_name`); so doing something like `@zero_hashes compute_zero_hashes(...)` is enough.

Reply:
Here: https://github.com/ralexstokes/ssz-rs/blob/1f94d5dfc70c86dab672e91ac46af04a5f96c342/ssz-rs/src/merkleization/mod.rs#L144

It's generated at compile time here.

Made an update. Compilation is slower rn though

Reply:
This seems to be where it's computed:
https://github.com/ralexstokes/ssz-rs/blob/1f94d5dfc70c86dab672e91ac46af04a5f96c342/ssz-rs/build.rs#L14-L24

Let's try to add that to the `ZeroHashes` module. If the compilation turns out to be too slow, we can just go back to the pre-computed hashes, and leave the way to compute them as a comment.

Reply:
That function is only called once when the project is built. And, the point of this is so that we don't need to do the hashing again and that what the function does.

See another example:

https://github.com/gballet/ssz.zig/blob/master/src/zeros.zig

Reply:
Yes. I'm just saying it might be useful as documentation. Both of those implementations have a function to compute those hashes. If we add that, future maintainers will more easily know what this magic string is. Also, just to clarify: as long as the module doesn't use any external functions, it won't need to be recompiled, so the code will run only the first time it's compiled (the same way `ssz-rs`'s implementation works).

Reply:
Gotcha.

Reply:
Added the function